NativeCore/ReClassNET_Plugin.hpp

```diff
@@ -7,6 +9,12 @@
 #include <locale>
 #include <cstring>
 
+// OS Specific
+
+#ifdef __linux__
+	#define __stdcall
```

Comment:
Is there a reason for this define? `__stdcall` should not be present in the linux files.

Reply:
Ok, this change is because of your changes to the sample plugin. I think it would be better to split the native sample plugin in a Windows and Linux project.

Reply:
__stdcall isnt present in gcc and since linux only really uses stdcall its not needed to supply the calling convention. however we could change it to

	#define __stdcall __attribute__((__stdcall__))

which would result in some compiler warnings

dllmain.cpp:12:86: warning: ‘stdcall’ attribute ignored [-Wattributes]